device-registry/src/main/scala/org/genivi/sota/device_registry/Routes.scala

```diff
@@ -120,54 +119,50 @@ class Routes(namespaceExtractor: Directive1[Namespace],
   implicit val NamespaceUnmarshaller: FromStringUnmarshaller[Namespace] = Unmarshaller.strict(Namespace.apply)
 
   def api: Route =
-    ErrorHandler.handleErrors {
```

Comment:
I moved it outside, to where the groups are joined together, in Boot.scala and ResourceSpec.scala.